acl/validation.go

```diff
@@ -3,17 +3,21 @@
 
 package acl
 
-import "regexp"
+import (
+	"regexp"
+	"strings"
+)
 
 const (
 	ServiceIdentityNameMaxLength = 256
 	NodeIdentityNameMaxLength    = 256
+	PolicyNameMaxLength          = 128
 )
 
 var (
 	validServiceIdentityName = regexp.MustCompile(`^[a-z0-9]([a-z0-9\-_]*[a-z0-9])?$`)
 	validNodeIdentityName    = regexp.MustCompile(`^[a-z0-9]([a-z0-9\-_]*[a-z0-9])?$`)
-	validPolicyName          = regexp.MustCompile(`^[A-Za-z0-9\-_]{1,128}$`)
+	validPolicyName          = regexp.MustCompile(`^[A-Za-z0-9\-_]+/?[A-Za-z0-9\-_]*$`)
```

Comment:
nit suggestion - I know the other regex don't have this but I think it could be helpful to leave a comment on what this regex intends to match on. No strong feelings, leave it up to you!

Can you check if my understanding of the regex is right?
1. start with at least one [alphanumeric, "-", or "_" character]
2. in the middle, permit to have one "/" 
3. then end with at least one [alphanumeric, "-", or "_" character]

As a side note, when I inputted this into https://regexr.com, and it warns that the forward slash is unescaped. Would it make sense to change `/` to `\/`?

Reply:
It depends on the regex engine. Python and Go don't seem to care if it's escaped or not, but PCRE and JavaScript do. I can escape the slash though.

Reply:
Ah, good to know. I'm okay either way then and will leave it up to you. Thanks!